grafonnet-base/helpers.libsonnet

```diff
@@ -40,11 +47,20 @@ local xtd = import 'github.com/jsonnet-libs/xtd/main.libsonnet';
       { [root.last(splitPath)]:: {} }
     ),
 
+  // Transform moves the content from JSONPath `from` to JSONPath `to` in `source`
```

Comment:
This isn't so much transforming as extending? 'Cos you don't remove the old.

Reply:
Note that I expect these helpers to be refactored in the near future, the bespoke groupings/mapping format should eventually be formalized upstream and then these functions should accomodate that instead of implementing their own logic. 

So I see naming and limited documentation not much as a problem considering the temporary nature of this.